### PR TITLE
Remove any reference to protos and GRPC

### LINKED ISSUE #1632: Remove any reference to protos and GRPC
Fybrik no longer supports grpc communication with connectors. Thus, any reference that we have in the code to proto files, protoc, grpc files can be removed.

### PR DESCRIPTION
**Pull request by revit13**
/Closes #1632

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -221,6 +220,7 @@ require (
 	gomodules.xyz/jsonpatch/v2 v2.2.0 // indirect
 	google.golang.org/appengine v1.6.7 // indirect
 	google.golang.org/genproto v0.0.0-20220107163113-42d7afdf6368 // indirect
+	google.golang.org/protobuf v1.27.1 // indirect
```
Comment (shlomitk1) [inline go.mod:223]: Do we need genproto and protobuf packages?

Comment (revit13) [reply go.mod:223]: I tried to remove that and it complains so I guess it is a dependency with other modules.

Comment (shlomitk1) [reply go.mod:223]: Probably because of grpc in the notebook read/write tests

**Review by shlomitk1 — DISMISSED** [review]

**Review by roytman — COMMENTED** [review]
- please update line 21 in site/Readme.md.
- the file site/docs/reference/connectors.md talks about protobuf connectors

**Comment by revit13** [comment]
> * please update line 21 in site/Readme.md.
> * the file site/docs/reference/connectors.md talks about protobuf connectors

Done, Thanks

**Review by roytman — APPROVED** [review]

**Pull request merged**